lib/elixir/test/elixir/kernel/dialyzer_test.exs

```diff
@@ -54,11 +54,11 @@ defmodule Kernel.DialyzerTest do
     {:ok, [outdir: dir, dialyzer: dialyzer]}
   end
 
-  @tag otp19: false
-  test "no warnings on valid remote calls", context do
-    copy_beam! context, Dialyzer.RemoteCall
-    assert_dialyze_no_warnings! context
-  end
+  # @tag otp19: false
```

Comment:
@michalmuskala oh, I'm sorry. I missed that. :)

I think we should probably disable this using a new tag or delete the test. I would lean toward the former.